src/pmproxy/src/server.c

```diff
@@ -767,6 +737,7 @@ shutdown_ports(void *arg)
     }
 
     uv_loop_close(proxy->events);
+    pmWebTimerReleaseAll();
```

Comment:
@goodwinos  doesn't this have to be called before uv_loop_close?  (on the previous line) ... else how can the loop exit cleanly, with timers active timers.